doc/args_nonci.rst

```diff
@@ -0,0 +1,21 @@
+:orphan:
+
+Non-CI subcommand
+-------------------
+
```

Comment:
I still don't see a direct answer to the question 'What is this?'

Reply:
Sorry, I don't know what I can add here to fulfil all requirements